### PR TITLE
App Routes

### PR DESCRIPTION
**Pull request by bvenceslas**

### CODE REVIEW

File: models/maison.model.js [hunk 1]
```diff
@@ -3,16 +3,15 @@ const UUID = require("uuid");
...
-        ville: {type: String, required: [true, 'ville  ne doit pas etre vide']},
-        quartier: {type: String, required: [true, 'quartier  ne doit pas etre vide']},
-        avenue: {type: String, required: [true, 'avenue  ne doit pas etre vide']},
-        numero: {type: Number, required: [true, 'avenue  ne doit pas etre vide']}
-    },
+    province: {type: String, required: [true, 'province ne doit pas être vide']},
+    ville: {type: String, required: [true, 'ville  ne doit pas être vide']},
+    quartier: {type: String, required: [true, 'quartier  ne doit pas être vide']},
+    avenue: {type: String, required: [true, 'avenue  ne doit pas être vide']},
+    numero: {type: Number, required: [true, 'avenue  ne doit pas être vide']},
```
Comment (guillainbisimwa) [inline models/maison.model.js]: The number of the **avenue** `numero` may or may not be compulsory!

File: models/maison.model.js [hunk 2]
```diff
@@ -3,16 +3,15 @@ const UUID = require("uuid");
 
 const Maison = mongoose.Schema({
```
Comment (guillainbisimwa) [inline models/maison.model.js:5]: Also, add latitude and longitude. These are `floats`.

File: models/produit.model.js [hunk 3]
```diff
@@ -2,15 +2,15 @@ const mongoose = require('mongoose');
 
 const Produit = mongoose.Schema({
```
Comment (guillainbisimwa) [inline models/produit.model.js:3]: We can also have the characteristics of the products!

- [ ] Color
- [ ] Weight
- [ ] Guaranteed
- [ ] Other characteristics

EX: Products like clothes, chocolate (white or dark, ...)

**Review by guillainbisimwa — COMMENTED** [review]
## Hi again    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
we can also have the characteristics of the products!

# \[OPTIONAL\] 
Important to have a Geographic coordinate system!

### Make changes and submit another code review request.
*Happy coding*

File: models/produit.model.js (hunk 3 above)
Comment (bvenceslas) [reply models/produit.model.js:3]: noted, let me work on it

File: models/maison.model.js (hunk 2 above)
Comment (bvenceslas) [reply models/maison.model.js:5]: noted

File: models/maison.model.js (hunk 1 above)
Comment (bvenceslas) [reply models/maison.model.js]: noted

**Review by guillainbisimwa — APPROVED** [review]
# Approved

Thank you for making the required changes, your PR is now approved!

### Please, don't forget to merge to main!

Best.

**Pull request merged**